lib/asciidoctor/cc/fonts_manifest.yaml

```diff
@@ -0,0 +1,5 @@
+---
+SourceSansPro:
```

Comment:
Don’t we need the full font names here? Ping @alexeymorozov

Reply:
Also, please move this file under lib/metanorma/cc. It has nothing to do with Asciidoctor which is a parser.